source/features/clean-conversation-headers.tsx

```diff
@@ -24,23 +25,23 @@ function initPR(): void {
 	observe('.gh-header-meta .flex-auto:not(.rgh-clean-conversation-header)', {
 		add(byline) {
 			byline.classList.add('rgh-clean-conversation-header');
-			const isMerged = select.exists('#partial-discussion-header [title="Status: Merged"]');
 			const isSameAuthor = select('.js-discussion > .TimelineItem:first-child .author')?.textContent === select('.author', byline)!.textContent;
 			const baseBranch = select('.commit-ref:not(.head-ref)', byline)!;
 			const isDefaultBranch = (baseBranch.firstElementChild as HTMLAnchorElement).pathname.split('/').length === 3;
+			const prNotOpen = isClosedPR() || isMergedPR();
```

Comment:
Don't like the var name here. What would be a better name?

Reply:
Merged PR are also closed PRs. Just use a single isClosedPR detection instead of 2

Reply:
But I still need a separate is merged. 

So make a isMerged and isClosed should include is merged right?